### DISCUSSION TITLE
Where is the Gerber data for the NGP adapter?

### DISCUSSION THREAD

**Discussion by gatarib** [sanni/cartreader#614] [Need help]
I was going to make an NGP adapter but can't find the gerber data. Where can I find it?

**Comment by MattroidPrime** [comment]
I was hoping to find the same but haven't had any luck. [This is the best I've found](https://savethehero.builders/collections/cartridge-adapters/products/ngp-adapter-only-pcb-board ) that is actually available and of course still needs a slot. I still have my old [Linkmasta dumper](https://www.flashmasta.com/hardware-info/neo-geo-pocket-linkmasta/) but it'd be nice to have everything in one device. 

For anyone else who comes across this thread and is wondering about the connector (without ripping one out of what is now a pretty expensive handheld, even for parts), [here's an old guide on using a PCI socket](https://www.flashmasta.com/how-to-make-a-videogame-cartridge-connector-for-neo-geo-pocket-or-gameboy-nes-snes-genesis-etc/).

**Comment by Daze909** [comment]
it's here
https://github.com/sanni/cartreader/tree/bd3eaa106b6e4cbdffbfa89a3fb5fedf3e028b17/pcb/adapters

I made it by processing SSCI-034173

**Reply by gatarib** [reply]
Thank you! I'll give it a shot too!

**Reply by sanni** [reply]
Is it this connector? https://www.adafruit.com/product/3342
Datasheet: https://cdn-shop.adafruit.com/product-files/3342/Micro-Bit_connector_datasheet.pdf

**Reply by Ancyker** [reply]
Here's another candidate: https://www.digikey.com/en/products/detail/sullins-connector-solutions/RBB40DHHN/927341

**Reply by Ancyker** [reply]
... and another: https://www.digikey.com/en/products/detail/pi-supply/PIS-1240/10315765

Edit: This one appears to be for the same board @sanni linked. The data sheet makes it look like an extremely good candidate.

**Reply by MattroidPrime** [reply]
Thanks for this! I thought I'd circle back around on this and confirm that the connector @sanni linked works great, with some necessary modifications. A bit of work with a dremel and hot knife and the result, while not pretty, is great. 

![NGP_adapter](https://user-images.githubusercontent.com/70599385/207782242-1394a67b-cef9-4ae0-ab67-1d2c45f24d92.jpg)

![NGP_adapter_socket](https://user-images.githubusercontent.com/70599385/207782296-b09521a1-5fb9-486a-9e4a-90502544aa1e.jpg)

**Reply by agammell** [reply]
Just a note for anyone coming to this from the adapters link: the connector linked above that's DigiKey part number 1910-PIS-1240-ND is not the same as the adafruit link & does **_not_** work. It's slightly the wrong size to mount on the board easily and is only a single row of pins rather than the required double row.

(You might be able to get it to work with a lot of manual straightening & then bending of the pins back to the correct length & I may eventually try that. But it will be a pain.)

**Reply by jayhamm7** [reply]
> Just a note for anyone coming to this from the adapters link: the connector linked above that's DigiKey part number 1910-PIS-1240-ND is not the same as the adafruit link & does **_not_** work. It's slightly the wrong size to mount on the board easily and is only a single row of pins rather than the required double row.
> 
> (You might be able to get it to work with a lot of manual straightening & then bending of the pins back to the correct length & I may eventually try that. But it will be a pain.)

Thank you for this comment. Maybe others would have noticed the difference but I wasn't thinking that hard. I found an alternative seller to the original adafruit link that I'm going to try
 
https://www.pishop.us/product/2x40-right-angle-edge-connector-for-microbit/
